src/commands/feedback.js

```diff
@@ -1,6 +1,57 @@
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
 
+async function collector(interaction, message, embed) {
+
+	const filterReaction = (reaction, user) => {
```

Comment:
```suggestion
	const filterReaction = (user) => {
```
Since it won't check the reaction, just remove from the params.